Source/WebCore/Modules/highlight/Highlight.cpp

```diff
@@ -48,6 +48,13 @@ static void repaintRange(const AbstractRange& range)
     }
 }
 
+static bool repaintIfPriorityChanged(int& oldPriority, int& newPriority)
```

Comment:
`int` is a POD type, just pass by value

```suggestion
static bool repaintIfPriorityChanged(int oldPriority, int newPriority)
```